client/my-sites/earn/launchpad/index.tsx

```diff
@@ -58,7 +32,11 @@ const EarnLaunchpad = ( {
 					{ translate( 'Let your fans support your art, writing, or project directly.' ) }
 				</p>
 			</div>
-			<Launchpad siteSlug={ site?.slug ?? null } checklistSlug="earn" taskFilter={ taskFilter } />
+			<DefaultWiredLaunchpad
+				siteSlug={ site?.slug ?? null }
+				checklistSlug="earn"
+				launchpadContext="earn"
+			/>
```

Comment:
Updating from Launchpad to DefaultWiredLaunchpad here allows us to remove the extra code around the taskFilter.